docs/methods.rst

```diff
@@ -1174,7 +1174,7 @@ Additional information and recommendations:
    parameters discussed earlier for descent are useful for setting up
    the descent engine to do this averaging on its own.
 
-Stochastic Reconfiguration
+Stochastic Reconfiguration with Conjugate Gradient
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
also need to add enough `^^^` otherwise we get warnings when compiling the docs.

Reply:
again moving too fast